.changes/next-release/bugfix-eks-27483.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "eks get-token",
```

Comment:
This was based on @kyleknap 's comment https://github.com/aws/aws-cli/pull/6476#discussion_r866192743

I can change it to just "eks"

Reply:
I'll defer to Kyle then if that's the category he wants.

Reply:
I added another as a feature one that is also `eks get-token`. I'm fine with that as the category.

Reply:
I think `eks get-token` make sense especially because it is a custom command. One super nitpick thing that I missed in the last PR that I reviewed was that we should encompass the category with double backticks so it shows up as a code literal in the change log (e.g. "\`\`eks get-token\`\`" instead of just "eks get-token"). We should make sure to do that for both change logs entries we have now.